src/ck3/tables/include/data_global_functions.rs

```diff
@@ -621,4 +629,1515 @@
     ("UsesTimerLocking", Args::Args(&[]), bool),
     ("WatchWindowsEnabled", Args::Args(&[]), bool),
     ("WeeksPerYear", Args::Args(&[]), int32),
+    ("abandon_celibacy", Args::Args(&[]), Unknown),
```

Comment:
All these lowercase ones need to be removed; they are gameconcepts, which are handled separately because mods can change them.